web/src/components/molecules/Content/types.ts

```diff
@@ -0,0 +1,19 @@
+import { FieldType } from "@reearth-cms/components/molecules/Schema/types";
+
+export type ItemField = {
+  schemaFieldID: string;
+  type: FieldType;
+  value: any;
+};
+
+export type Item = {
+  id: string;
+  schemaID: string;
+  fields: ItemField[] | undefined | null;
+};
+
+export type ContentTableField = {
+  id: string;
+  schemaID: string;
+  fields: { [key: string]: any };
```

Comment:
Can the value actually be anything? Or can you type this better?

Reply:
@KaWaite , yes it can be anything
![1](https://user-images.githubusercontent.com/78056580/194231012-6d32b23b-c078-4054-bfb1-f722d46eea42.png)

Reply:
ok!